Fixed typos and added reference to docker-compose installation.

nisevi opened the pull request:
Fixed typos and added a reference to docker-compose installation (mostly for Linux users).

nisevi committed:
Misspelled word.

nisevi committed:
Added reference to docker-compose installation.

nisevi committed:
Fix typo.

gvilarino commented on 3-docker-compose/README.md:
```diff
@@ -2,7 +2,7 @@
 While docker allows you to easily build and link different apps with different environments and runtimes, doing so with more than just two or three containers quickly becomes tedious. `docker-compose` allows you to work with complex container systems very easily.
 
-For working with `docker-compose`, you will need a `docker-compose.yml` file. Check out the one under the `gvilarino/docker-testing` repo.
+For working with `docker-compose` you will need to [install it](https://docs.docker.com/compose/install/) depending on your case and you will need a `docker-compose.yml` file. Check out the one under the `gvilarino/docker-testing` repo.
```
I'd do something like:

First check if you have `docker-compose` installed (chances are that you have) by doing:

```
docker-compose --version
```

If that fails, [get `docker-compose` for you specific setup](https://docs.docker.com/compose/install/)

gvilarino commented:
Thanks! I left some comments, please address those and I'll merge.

nisevi committed:
Revert "Added reference to docker-compose installation."

This reverts commit 53209540c915b1f7e90b94860c5c1406b4e5c61d.

Reverted commit due to requested changes.

nisevi committed:
Changed wording as requested.

Removed 'First' from line 7 since is being used before, for describing how to check the `docker-compose` version.

nisevi commented on 3-docker-compose/README.md:
```diff
@@ -2,9 +2,17 @@
+
 For working with `docker-compose`, you will need a `docker-compose.yml` file. Check out the one under the `gvilarino/docker-testing` repo.
 
-First make sure you aren't running any containers. The from the project root, do:
```
@gvilarino I removed the word 'First' here since we are using it on line 5 now.

gvilarino reviewed (approved):
Awsome
